Gruntfile.js

```diff
@@ -148,6 +153,7 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-contrib-uglify');
   grunt.loadNpmTasks('grunt-shell');
   grunt.loadNpmTasks('grunt-ts');
+  grunt.loadNpmTasks('grunt-tslint');
```

Comment:
How does one kick off a `tslint` run given this config? I expected a build target or npm script to be required.

Reply:
The additions within the `grunt.initConfig` section define the targets.  With these changes, `grunt tslint:src`, `grunt tslint:test`, and `grunt tslint:all` (all source) are all valid.